geoips/plugins/modules/output_checkers/image.py

```diff
@@ -22,6 +22,53 @@
 name = "image"
 
 
+def yield_test_files():
```

Comment:
It's now "get_test_files"

Reply:
Looks good, resolved here: https://github.com/NRLMMD-GEOIPS/geoips/pull/379/files#r1373549085

Reply:
I think that @AJThorpe used something else in the readers PR. I think it was `gen_test_files` rather than `get_test_files`. We should choose one and be consistent. I kind of lean towards `get_` but would be fine either way.

Reply:
Updated to get